packages/salesforcedx-vscode-soql/test/vscode-integration/sfdx.test.ts

```diff
@@ -0,0 +1,27 @@
+import { expect } from 'chai';
+import * as sinon from 'sinon';
+import * as vscode from 'vscode';
+import * as sfdx from '../../src/sfdx';
+
+describe('sfdx utils should', () => {
+  let sandbox: sinon.SinonSandbox;
+
+  beforeEach(() => {
+    sandbox = sinon.createSandbox();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  it('displays an error if can not get connection to org', async () => {
```

Comment:
This test should also verify you are sending data to the channel service

Reply:
and verified i'm sending telemetry as well.